deployment/ansible/roles/nyc-trees.celery/templates/upstart-nyc-trees-celery.conf.j2

```diff
@@ -0,0 +1,14 @@
+description	"nyc-trees-celery"
+
+{% if ['development', 'test'] | some_are_in(group_names) -%}
+start on (vagrant-mounted)
+{% else %}
+start on (local-filesystems and net-device-up IFACE!=lo)
+{% endif %}
+stop on shutdown
+
+respawn
+setuid nyc-trees
+chdir {{ app_home }}
+
+exec envdir /etc/nyc-trees.d/env celery -A "nyc_trees.celery:app" worker -l {{ celery_log_level }} >> {{ celery_log }} 2>&1
```

Comment:
I was under the impression that the preferred way of running celery with django is to use some form `python manage.py celery`. This might also obviate the need for `nyc_trees/celery.py`.